src/test/unittest/unittest.sh

```diff
@@ -38,6 +38,25 @@ export LC_ALL="C"
 
 . ../testconfig.sh
 
+function interactive_color() {
+	local color
+	color=$(shift)
+
+	if [ -t 1 ] && command -v tput >/dev/null; then
```

Comment:
Original code used different fds depending on whether message was supposed to go to stdout or stderr.